python/py-socksio/Portfile

```diff
@@ -20,9 +20,14 @@ checksums           rmd160  fb21d64a51249d690c7b9fb409c48b0a4113739b \
                     sha256  f88beb3da5b5c38b9890469de67d0cb0f9d494b78b106ca1845f96c10b91c4ac \
                     size    19055
 
-python.versions     37 38 39 310
+python.versions     37 38 39 310 311
 
```

Comment:
can this not use "python.pep517 yes" and then setting the "python.pep517_backend" appropriately? (Of course you probably still need the reinplace but I think shouldn't need to add (some of the) dependencies manually anymore.